src/Microsoft.Docs.LearnValidation/Utils/Utility.cs

```diff
@@ -7,19 +7,16 @@ namespace Microsoft.Docs.LearnValidation
 {
     public static class Utility
     {
-        private static readonly Regex s_sshGitUrlRegex = new(@"git@(?<host>.+?):(?<userName>.+?)\/(?<repoName>.+)\.git", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+        private static readonly Regex SshGitUrlRegex = new(@"git@(?<host>.+?):(?<userName>.+?)\/(?<repoName>.+)\.git", RegexOptions.Compiled | RegexOptions.IgnoreCase);
```

Comment:
revert this change: it should cause a build warning: https://dev.azure.com/ceapex/Engineering/_build/results?buildId=261090&view=logs&j=a29a2371-0793-5e14-39b9-20eb0beef0db&t=61e53b64-9dac-5231-7eb0-82834e091e5f&l=51

Reply:
ok